src/keyValue.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+#ifndef SRC_KEYVALUE_H_
+#define SRC_KEYVALUE_H_
+#include <unordered_map>
+#include <vector>
+
+namespace backend {
+
+// TODO: CHANGE BYTES TYPE
+typedef char BYTES;
+
+class KeyValue {
+ public:
+  KeyValue() {}
+
+  ~KeyValue() {}
+
+  void put(const BYTES& key, const BYTES& value);
```

Comment:
Capitalize method names.

Also, are these the right argument types?

Reply:
should I not use a typedef and just switch to string?

Reply:
You should either use string or use a template; it's better not to have a typedef type unless you really need it for convenience.

Reply:
ok makes sense